invenio_rdm_migrator/streams/github/transform.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# Invenio-RDM-Migrator is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Invenio RDM migration GtiHub transform interfaces."""
+
+from ...transform import Transform
+
+
+class GitHubReleaseTransform(Transform):
```

Comment:
You can be very explicit, as in this case, or use the `IdentityMixin` class.

Reply:
Yeah, I was wondering actually what the use was for, and it looks like this would be better. I don't have any tests unfortunately, but maybe when we add the actions, I'll give it a go.